ecosystem/indexer-grpc/indexer-grpc-fullnode/src/runtime.rs

```diff
@@ -205,3 +224,93 @@ pub fn get_status(
         chain_id: ledger_chain_id as u32,
     }
 }
+
+/// External service on the fullnode is for testing/local development only.
+/// Performance is not optimized, e.g., single-threaded.
+#[tonic::async_trait]
+impl RawData for RawDataService {
```

Comment:
let's extract this out to a different function?

Reply:
done.